internal/factsengine/gatherers/corosynccmapctl.go

```diff
@@ -45,6 +44,38 @@ func NewCorosyncCmapctlGatherer(executor utils.CommandExecutor) *CorosyncCmapctl
 	}
 }
 
+func corosyncCmapctlOutputToMap(corosyncCmapctlOutput string) *entities.FactValueMap {
+	outputMap := &entities.FactValueMap{Value: make(map[string]entities.FactValue)}
+	var cursor *entities.FactValueMap
+
+	for _, line := range strings.Split(corosyncCmapctlOutput, "\n") {
+		cursor = outputMap
+
+		value := strings.Split(line, "= ")[1]
+
+		pathAsString := strings.Split(line, " (")[0]
```

Comment:
Other thing. I have seen that some values might include parenthesis, as they are string.
We might want to run this 2nd split using the `value` string instead of the `line`

Reply:
Could you add a test on this?
The issue would come with this line: `runtime.totem.pg.mrp.srp.members.1.ip (str) = r(0) ip(10.80.1.11) `